crates/containerd-shim-wasmedge/src/executor.rs

```diff
@@ -118,6 +125,28 @@ fn env_to_wasi(spec: &Spec) -> Vec<String> {
     env.to_vec()
 }
 
+fn genereate_preopen(spec: &Spec) -> Vec<String> {
+    let mut preopens: Vec<String> = vec![];
+    preopens.push("/:.".to_string());
```

Comment:
Why is this moving back to CWD? 
 https://github.com/containerd/runwasi/commit/20900fc26c9f572d35c4368d50a0122ad4b413b6.

Reply:
You are correct. This feature has been delayed for a while, so I have rebased to the older version of the code. 
And I have noticed that the architecture seems to have changed again. I will fix it when I rebase it done.

Reply:
recover done.